demisto_sdk/commands/prepare_content/integration_script_unifier.py

```diff
@@ -552,10 +552,11 @@ def insert_pack_version(
             existing_pack_info_debug_log_re = (
                 r"demisto\.debug\('pack name = .*?, pack version = .*?'\)"
             )
-            global_name = f"{pack_id.upper()}_PACK_VERSION"
+            global_name = "PACK_VERSION"
+            pack_id_escaped = pack_id.replace("'", "\\'")
```

Comment:
what is this?

Reply:
This is a fix that Dan Tabori added, because if there were a ' in the name it would have caused a bug, and I removed it because I changed it to id